### PR TITLE
gives a player fortress points and displays a HUD message whenever a …

### PR DESCRIPTION
**Pull request by alaswell**
…player disables security

I've seen/heard this asked for a BUNCH of times so yeah now it's done :smiley_cat:

### CODE REVIEW

File: maps/includes/base_shutdown.lua [hunk 1]
```diff
@@ -88,6 +88,8 @@ function button_common:allowed( allowed_entity )
 	if IsPlayer( allowed_entity ) then
 		local player = CastToPlayer( allowed_entity )
 		if player:GetTeamId() == self.team and player:IsAlive() then
+			player:AddFortPoints(100, "disabled security")
```
Comment (squeek502) [inline maps/includes/base_shutdown.lua]: This should be a localized string (`#FF_FORTPOINTS_DISABLE_SECURITY` for example).

See https://github.com/fortressforever/fortressforever-scripts/blob/master/maps/includes/base_teamplay.lua#L431

**Comment by AfterShockFF** [comment]
Maybe 100 points is a bit much? I don't think its equivalent to killing a player. I guess it depends on the map but many maps its just a 5 second diversion. Maybe 25-50 would be more suitable?

**Comment by squeek502** [comment]
50 might be better, but 100 might not be too bad.

In a 15 minute round and with a security length of 30 seconds, security will be pressed a maximum of 30 times (or a max of 20 with a 45 second security length). In reality, it's probably much lower than that, but it also depends on the map (if the first toss gets the flag out of security, then the button would need to be pressed a lot less).

File: maps/includes/base_shutdown.lua (hunk 1 above)
Comment (squeek502) [inline maps/includes/base_shutdown.lua]: The 100 (or whatever number of points it ends up being) should also be abstracted out into a global variable (`FORTPOINTS_PER_SECURITY_DISABLE` for example).

**Comment by squeek502** [comment]
:+1: but it looks like you need to rebase in order to solve conflicts in FortressForever_english.txt.

**Pull request merged**